RELEASING.md

```diff
@@ -0,0 +1,96 @@
+PR should (ideally) be an empty commit, but might contain some minor changes if
+we want to get them in for a specific release. You can create an empty commit
+by running:
+```
+git commit --allow-empty
+```
+
+The description of the PR should just be the release notes that we want to
+publish in the GitHub Release. The notes should just have one-line summaries
+of the PRs in the release. Trivial PRs can be omitted and releated PRs can be
+combined in a single line. It should have the following subsections:
+  - "Breaking Changes" (backwards-incompatible changes to the package API)
+  - "New Features" (backwards-compatible changes to the package API)
+  - "Bug Fixes" (fixes to any particular issues)
+  - "Other Changes" (non-breaking code changes oor Doc/CI updates)
+
+Sections can be omitted if there wouldn't be any PRs under them. The
+[`v0.3.2` release notes](https://github.com/google/go-tpm-tools/releases/tag/v0.3.2)
+are a good example. We don't need to specifically mention who wrote what PR or
+link to the "Full Changelog". Users can just look this stuff up on GitHub on
+their own.
+
+This commit _should not_ change the version numbers in [`go.work`](go.work),
+[`cmd/go.mod`](cmd/go.mod), or [`launcher/go.mod`](launcher/go.mod). When
+reviewing the PR, the reviewers and author should decide if the release
+will be a major, minor, or patch release. Note that the PR should only consist
+of a single commit and be "squashed" instead of "merged".
+
+## Tag the releases
+
+After the new release is in the `master` branch, we need to create git tags so
+that the Go version system can find the releases. Generally the author of the
+PR should do this.
+
+Tagging can be done via the GitHub Web UI. On the
+[Releases Page](https://github.com/google/go-tpm-tools/releases),
+click [Draft a New Release](https://github.com/google/go-tpm-tools/releases/new).
+In that draft, create the git tag corresponding to your release, and copy the
+approved release notes into the description.
+
+Check that the preview of the release notes looks good, and click
+"Publish release". The release and tag should now be visible on GitHub. 
+
+## Follow-up Submodule update PR
+
+After the main release has been merged and tagged, we need to update the go.mod
+files in the various submodules. First, you should update the version number in:
+  - [`go.work`](go.work)
+  - [`cmd/go.mod`](cmd/go.mod)
+  - [`launcher/go.mod`](launcher/go.mod)
+
+Next, we cleanup the modules by running:
+  - run `go mod tidy` in each module directory
+  - run `go work sync` in the root directory
+  - this requires Go 1.18 or later
+
+Finally, create a PR with the title "Submodule update for vX.Y.Z". This PR
+doesn't need a description. The reviewers should just check that the above
+steps were done. Note that the PR should only consist
+of a single commit and be "squashed" instead of "merged".
+
+## Tagging the submodules
```

Comment:
Do we really want to update the submodules 1:1 with the root repo module? They are separate modules and may not always use features of the root repo